compiler/rustc_parse_format/src/lib.rs

```diff
@@ -38,6 +38,19 @@ impl InnerSpan {
     }
 }
 
+#[derive(Copy, Clone, PartialEq, Eq)]
+pub struct InnerWidthMapping {
+    pub position: usize,
+    pub before: usize,
+    pub after: usize,
```

Comment:
Could you comment the individual fields?
What  are they: positions? Starting at which point of reference?

Reply:
Added some comments.

For what it's worth, I think this could also be refactored to instead have a `InnerSpan` and a `usize`, which might make some of the intent more clear.